Remotion/Web/Core/UI/Controls/WebTabStripImplementation/Rendering/WebTabRenderer.cs

```diff
@@ -156,10 +147,9 @@ protected virtual Command RenderBeginTagForCommand (WebTabStripRenderingContext
       {
         command.Type = CommandType.Event;
 
-        var textWithHotkey = HotkeyParser.Parse (tab.Text);
-        if (textWithHotkey.Hotkey.HasValue)
-          command.AccessKey = _hotkeyFormatter.FormatHotkey (textWithHotkey);
-
+        if (string.IsNullOrEmpty (command.AccessKey))
+          attributes.Add (HtmlTextWriterAttribute.Accesskey.ToString(), command.AccessKey);
```

Comment:
Condition is wrong.

Reply:
`HtmlTextWriterAttribute.Accesskey.ToString()` vs `nameof (HtmlTextWriterAttribute.Accesskey)`